### ISSUE TITLE
Twodsix Compatibility error

### ISSUE THREAD

**Issue by zedsupremus** [mclemente/healthEstimate#203]
When used with with twodsix
Seems to throw 
logic.js:314 SyntaxError: Invalid or unexpected token
    at new Function (<anonymous>)
    at HealthEstimate.breakOverlayRender (logic.js:295:12)
    at addCharacter (HealthMonitor.js:34:45)
    at Array.forEach (<anonymous>)
    at Object.onCanvasReady [as fn] (hooks.js:31:10)
    at #call (foundry.js:730:20)
    at Hooks.call (foundry.js:712:38)
    at Canvas.draw (foundry.js:26630:11)
    at async Scene.view (foundry.js:23682:7)
    at async Game.initializeCanvas (foundry.js:8837:20)

System in question being here in estimation If I'm not mistaken,
https://github.com/mclemente/healthEstimate/blob/e54fbce610b7163d71c62bc87a09bf851ab82e64/module/EstimationProvider.js#L1501C2-L1501C2

**To Reproduce**
Have 2d6 system
Create actor/animal/vehicle
Drop on canvas. 
Inspect Console

**Game Specifics (please complete the following information):**
 - Twodsix 4.1.0 though its been happening for a long time
 - Health Estimate 31.6.2  though its been happening for a long time

**Additional context**
I -think- this began either in April or after V11. Its unclear as It was looking fine to a GM but broken for players. (And none of them mentioned it for months)
I have also reported this to Twodsix, its possible it may be related to a change for V11.

**Comment by zedsupremus** [comment]
Additionally looks like Ships, Space Objects and robots arent handled correctly (robots and space objects make sense, those are new).


logic.js:175 Health Estimate | Error on getEstimation(). Token Name: "Wasteful Test". Type: "ship". Error: Token's fraction is not valid, it probably doesn't have a numerical HP or Max HP value.
    at HealthEstimate.getFraction (logic.js:191:10)
    at HealthEstimate.getEstimation (logic.js:162:33)
    at HealthEstimate._handleOverlay (logic.js:63:42)
    at Object.refreshToken [as fn] (hooks.js:115:23)
    at #call (foundry.js:730:20)
    at Hooks.callAll (foundry.js:687:17)
    at Token.applyRenderFlags (foundry.js:28389:11)
    at #applyRenderFlags (foundry.js:27744:44)
    at nn.emit (index.js:89:1)
    at Dt.update (index.js:89:1)
    at Dt._tick (index.js:89:1)

logic.js:175 Health Estimate | Error on getEstimation(). Token Name: "Knowledgeable Test". Type: "robot". Error: Token's fraction is not valid, it probably doesn't have a numerical HP or Max HP value.
    at HealthEstimate.getFraction (logic.js:191:10)
    at HealthEstimate.getEstimation (logic.js:162:33)
    at HealthEstimate._handleOverlay (logic.js:63:42)
    at Object.refreshToken [as fn] (hooks.js:115:23)
    at #call (foundry.js:730:20)
    at Hooks.callAll (foundry.js:687:17)
    at Token.applyRenderFlags (foundry.js:28389:11)
    at #applyRenderFlags (foundry.js:27744:44)
    at nn.emit (index.js:89:1)
    at Dt.update (index.js:89:1)
    at Dt._tick (index.js:89:1)
    
    logic.js:175 Health Estimate | Error on getEstimation(). Token Name: "Receptive Test". Type: "space-object". Error: Token's fraction is not valid, it probably doesn't have a numerical HP or Max HP value.
    at HealthEstimate.getFraction (logic.js:191:10)
    at HealthEstimate.getEstimation (logic.js:162:33)
    at HealthEstimate._handleOverlay (logic.js:63:42)
    at Object.refreshToken [as fn] (hooks.js:115:23)
    at #call (foundry.js:730:20)
    at Hooks.callAll (foundry.js:687:17)
    at Token.applyRenderFlags (foundry.js:28389:11)
    at #applyRenderFlags (foundry.js:27744:44)
    at nn.emit (index.js:89:1)
    at Dt.update (index.js:89:1)
    at Dt._tick (index.js:89:1)

**Comment by mclemente** [comment]
I've fixed the original issue.

As for the second issue:
- Robots and Space Objects didn't exist the last time the system's provider was created. I've added them now.
- Ships: Any token with 0 current HP and 0 max HP will return as invalid since it will result in a 0/0 division (NaN).
  - In these cases, the error is pretty much a "you forgot to add Max HP on this token" warning, which can be disabled by the "Hide Estimates on Tokens with 0 Max HP" setting.
